lighthouse-core/scripts/legacy-javascript/summary-sizes.txt

```diff
@@ -85,26 +57,11 @@ core-js-2-only-polyfill
    3504        es6-function-name/main.bundle.min.js
 
 core-js-2-preset-env-esmodules
-  2857         false/main.bundle.min.js
-  1352         true/main.bundle.min.js
+  188400       false/main.bundle.min.js
+   64217       true/main.bundle.min.js
+   63536       true-and-bugfixes/main.bundle.min.js
```

Comment:
The small difference from `bugfixes: true` is because it only reduces the number of transforms applied. And the `main.js` file used is super small.

Reply:
oh, and this is orthogonal from all this, I just wanted to add this.